docs/notes-about-airflow-nginx.md

```diff
@@ -0,0 +1,19 @@
+# Notes about Airflow on nginx
```

Comment:
These are from the nginx help file.. .why do we need this here?

Reply:
I moved to _nginx_ folder. Is this a good place for this?

Reply:
Can you summarize what this file is used for?

Reply:
https://github.com/avdata99/ckan-cloud-docker/blob/master/nginx/notes-about-airflow-nginx.md
in the _nginx_ folder

Reply:
Not what's in the file (that I can see) - what is this file used for? Why is it here?

Reply:
I deleted the file.